toc/working-groups/vulnerability-management.md

```diff
@@ -0,0 +1,40 @@
+# Vulnerability Management: Working Group Charter
+
+## Mission
+
+Provides discreet management of security vulnerabilities issues relevant for active CF projects.
+
+## Goals
+
+* Provides a single point of contact for security vulnerability reporting and management
+* Provides management of security vulnerability reports through to resolution, including but not limited to triage, reporter and team coordination, embargo negotiation, CVSS scoring, CVE assignments, pre-disclosure and disclosure
+
+## Scope
+
+* Triage incoming security vulnerability reports to [security@cloudfoundry.org](mailto:security@cloudfoundry.org)
+* Management of vulnerabilities through dedicated slack channels
+* When appropriate, negotiating suitable embargo periods with the reporter to afford component teams time to fix the issue before it becomes known publicly
+* When appropriate, assignment of CVE numbers to vulnerabilities/fixes
+* Publication of pre-disclosures to allow all CF editions time to adopt fixes for high/critical vulnerabilities before they become known publicly
+* Publication of disclosures
+
+## Non-Goals
+
+* Not adding security related features to Cloud Foundry
+
+## Technical Lead(s): 
+- Paul Warren
+
+## Execution Lead(s):
+- Monty Ijzerman
+- Khushboo Soni
+- Matthias Essenpres 
```

Comment:
I think this is the correct spelling of this person's name, based on past emails, if I'm thinking of the right person.
```suggestion
- Mathias Essenpries 
```